keyboards/3.0/en-t-k0-teletype.xml

```diff
@@ -28,19 +28,19 @@
     <key id="slash" to="/" />
   </keys>
 
-  <layerMaps form="hardware">
-    <layerMap id="base">
+  <layers form="hardware" hardware="us">
+    <layer id="base">
       <row keys="Q W E R T Y U I O P gap2" />
       <row keys="gap A S D F G H J K L enter gap" />
       <row keys="figs Z X C B V N M ltrs linefeed null" />
       <row keys="gap space gap" />
-    </layerMap>
-    <layerMap id="figs">
+    </layer>
+    <layer id="figs">
```

Comment:
There is no `figs` layer on a hardware keyboard.

Reply:
I'm thinking i might retire this teletype keyboard.